utils/utils.go

```diff
@@ -130,11 +131,22 @@ func PrepareLocalPathForUpload(localPath string, useRegExp bool) string {
 		localPath = localPath[3:]
 	}
 	if !useRegExp {
-		localPath = pathToRegExp(localPath)
+		localPath = pathToRegExp(cleanPath(localPath))
 	}
 	return localPath
 }
 
+// Clean /../ | /./ and return the double backslash in case of windows path
+func cleanPath(path string) string {
+	temp := path[len(path)-1:]
+	path = filepath.Clean(path)
+	if temp == `\` || temp == "/" {
+		path += temp
+	}
+	path = strings.Replace(path, `\`, `\\`, -1)
```

Comment:
Let's add the following comment to describe this line:
// Since filepath.Clean replaces \\ with \, we revert this action.